legacy_publisher/dita_publisher.py

```diff
@@ -79,14 +77,46 @@ def publish_regions(regions=None, sourcepath=None):
     isDestExist = os.path.exists(dirname(dita_dir+new_path))
     if not isDestExist:
         os.makedirs(dirname(dita_dir+new_path))
-    copy_command = 'cp '+source_path+new_path+' '+dita_dir+new_path
-    os.system(copy_command)
+    os.system('cp '+source_path+new_path+' '+dita_dir+new_path)
 
     with open(DITA_REGIONS_EXPORT_FILE, "w") as f:
        f.write(xml_string_with_doctype) 
 
```

Comment:
in the next block, the validation of the regions code has been removed. We need it.